### DISCUSSION TITLE
how to use reown-kotlin

### DISCUSSION THREAD

**Discussion by factblock-roy** [WalletConnect/walletconnect-monorepo#5387] [Developer Support]
Hello, I'm an app developer. Our app was developed in the form of web view, and I'm trying to log in from the app.

What I want is that when I press the button, the wallet connect modal comes out, and after selecting, I bring the signature and message through the authentication process. I'm currently using the kotlin appkit. I also succeeded in initializing it. The problem is that the modal appears, but no ui appears. Web3Button and NetworkButton also keep saying connecting. Is there any way I can do it.

Currently my project id is 1e7e4bba5b0c2721eb93310ae0ff511d

**Comment by factblock-roy** [comment]
current docs so weak guide

**Comment by rtomas** [comment]
cc @jakubuid
